scripts/run_tracker.py

```diff
@@ -87,7 +87,7 @@ def is_transferred(run, transfer_file):
                 #Rows have two columns: run and transfer date
                 if row[0] == os.path.basename(run):
                     return True
-        if os.path.exists(os.path.join(run, 'transferring')):
+        if os.path.exists(os.path.join(run, '.transferring')):
```

Comment:
hm, maybe, didn't think on that, and it may actually be useful to see if a run is being transferred. I'll change this! Tack!